data.bloodpac.org/dashboard/Public/open-links/BPDC000118/index.html

```diff
@@ -0,0 +1,108 @@
+    >
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+  <meta name="apple-mobile-web-app-capable" content="yes" />
+
+  <link href="./app.css" rel="stylesheet" />
+
+</head>
+<body>
+  <header>
+    <div class="row">
+      <div class="logo-row">
+        <img src="./blood-pac.img" alt="logo" class="logo"></img>
+      </div>
+    </div>
+  </header>
+  <br>
+  <h1>Study Accession ID BPDC000118</h1>
+  <br>
+  <h1>Establishing the Clinical Utility of ctDNA Analysis for Diag-nosis, Prognosis, and Treatment Monitoring of Retinoblastoma: The Aqueous Humor Liquid Biopsy</h1>
+  <br>
+  <h2>Abstract</h2>
+
+  <p>Because direct tumor biopsy is prohibited for retinoblastoma (RB), eye-specific molecular biomarkers are not used in clinical practice for RB. Recently, we demonstrated that the aqueous humor (AH) is a rich liquid biopsy source of cell-free tumor DNA. Herein, we detail clinically-relevant molecular biomarkers from the first year of prospective validation data. Seven eyes from 6 RB patients who had AH sampled at diagnosis and throughout therapy with ≥12 months of follow-up were included. Cell-free DNA (cfDNA) from each sample was isolated and sequenced to assess genome-wide somatic copy number alterations (SCNAs), followed by targeted resequencing for pathogenic variants using a RB1 and MYCN custom hybridization panel. Tumoral genomic information was detected in 100% of diagnostic AH samples. Of the seven diagnostic AH samples, 5/7 were positive for RB SCNAs. Mutational analysis identified RB1 variants in 5/7 AH samples, including the 2 samples in which no SCNAs were detected. Two eyes failed therapy and required enucleation; both had poor prognostic biomarkers (chromosome 6p gain or MYCN amplification) present in the AH at the time of diagnosis. In the context of previously established pre-analytical, analytical, and clinical validity, this provides evidence for larger, prospective studies to further establish the clinical utility of the AH liquid biopsy and its applications to precision oncology for RB.</p>
+
+  <table class="dcf-table dcf-table-responsive dcf-table-bordered dcf-table-striped dcf-w-100%">
+	<tbody>
+		<tr>
+			<td>Authors</td>
+			<td><span>Liya Xu<sup>1,2,†</sup>, Mary E. Kim<sup>1,3,†</sup>, Ashley Polski<sup>1,3</sup>, Rishvanth K. Prabakar<sup>4</sup>, Lishuang Shen<sup>5</sup>, Chen-Ching Peng<sup>1,2</sup>, Mark W. Reid<sup>1</sup>, Patricia Chévez-Barrios<sup>6</sup>, Jonathan W. Kim<sup>1,3</sup>, Rachana Shah<sup>7</sup>, Rima Jubran<sup>7</sup>, Peter Kuhn<sup>2,8,9,10</sup>, David Cobrinik<sup>1,3,8,11,12</sup>, Jaclyn A. Biegel<sup>5,13</sup>, Xiaowu Gai<sup>5,13</sup>, James Hicks<sup>2,8,11<\sup>, and Jesse L. Berry<sup>1,3,8,12,*<\sup>
+            <br>
+            L. Xu and M. Kim contributed equally as co-first authors of this article.
+            *corresponding author
+        </tr>
+		<tr>
+			<td>Affiliations</td>
```

Comment:
kind of weird to have superscript text in authors but no `Affiliations`?

Reply:
i will add those later, we just need to get the page up with the correct ID right now